client/root.js

```diff
@@ -50,12 +51,23 @@ async function handleLoggedIn( context ) {
 	const state = context.store.getState();
 	const siteSlug = getSiteSlug( state, primarySiteId );
 	const isCustomerHomeEnabled = canCurrentUserUseCustomerHome( state, primarySiteId );
+	const launchpadTasksComplete = false;
+	const launchpadFlow = 'newsletter';
 
 	let redirectPath;
 
 	if ( ! siteSlug ) {
 		// there is no primary site or the site info couldn't be fetched. Redirect to Reader.
 		redirectPath = '/read';
+	} else if (
+		isCustomerHomeEnabled &&
+		! launchpadTasksComplete &&
+		config.isEnabled( 'signup/launchpad' )
+	) {
+		// The new stepper launchpad onboarding flow isn't registered within the "page"
+		// client-side router, so page.redirect won't work. We need to use the
```

Comment:
Would it make sense to register it with page?

Reply:
Page is a client side router, so, if I understand things correctly, that would effectively mean rendering the launchpad again outside of the context of stepper.

The end result would be that we render launchpad two separate times -- once in the stepper flow and again in the page client side router. I'm not even entirely certain we could port the launchpad into `page` without reimplementing a significant amount of plumbing with it. That's all to say that it'd be way, way easier to just redirect to the stepper flow.